src/main/java/org/broadinstitute/consent/http/resources/DaaResource.java

```diff
@@ -430,4 +430,24 @@ public Response removeDacDaaRelationship(
       return createExceptionResponse(e);
     }
   }
+
+  @POST
+  @RolesAllowed({ADMIN, CHAIRPERSON})
+  @Path("{dacId}/upload/{oldDaaId}/{newDaaName}")
```

Comment:
I think I'd prefer to use IDs here instead of name strings. If we did use IDs, we should be able to get the name from the previous DAA.
```suggestion
  @Path("{dacId}/upload/{oldDaaId}/{newDaaId}")
```
I'm also on the fence for the word `upload`. In this scenario, the upload has already happened. What we're doing here is pure messaging. Maybe something like `migrated`, `updated`, or `moved`?

Reply:
We could also verify that the old and new DAAs really exist before sending the email.

Reply:
My main concern in using the newDaaId is that when this email endpoint will be getting called is at the same time that the endpoint that is creating this new DAA. So, we wouldn't necessarily have the newDaaId at the time of calling this endpoint because all of those endpoints get called upon save of the update to the DAC. 

Additionally, I believe we _do_ verify that the old DAA exists before sending the email in the [DAAService method](https://github.com/DataBiosphere/consent/pull/2340/files#diff-9ca6884b52d6c055b2f5b979f6a62a840fddde450921c232b8f5ad622abdc902R146), but again, since the new DAA is not created yet, we won't be able to check if the new DAA exists before sending the email.